jetbrains-core/tst-213/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.explorer
+
+import com.intellij.ide.projectView.PresentationData
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.application.runInEdt
+import com.intellij.openapi.util.Ref
+import com.intellij.testFramework.DisposableRule
+import com.intellij.testFramework.ExtensionTestUtil
+import com.intellij.testFramework.ProjectRule
+import com.intellij.ui.tree.AsyncTreeModel
+import com.intellij.ui.tree.StructureTreeModel
+import com.intellij.ui.treeStructure.Tree
+import com.intellij.util.concurrency.Invoker
+import com.intellij.util.ui.tree.TreeUtil
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.kotlin.any
+import org.mockito.kotlin.atLeastOnce
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import software.aws.toolkits.jetbrains.core.MockResourceCacheRule
+import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerNode
+import software.aws.toolkits.jetbrains.core.fillResourceCache
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import javax.swing.tree.TreeModel
+
+class AwsExplorerNodeProcessorTest {
```

Comment:
why is this whole file pulled out? is this another one that could be paired down?

Reply:
Yes, actually, the change can be gotten through a new file, added the change